tilelang/carver/matmul_analysis.py

```
@@ -521,10 +521,10 @@ def normalize_to_matmul(sch: tir.Schedule,
 def get_tensorized_func_and_tags(
     func: tir.PrimFunc,
     target: Target,
-    layout: Optional[List[str]] = None,
+    layout: list[str] | None = None,
     skip_normalize: bool = False,
     allow_gemv: bool = False,
```

Ruff (0.14.1)

Unused function argument: `allow_gemv`

(ARG001)